### DISCUSSION TITLE
What's this issue template for?

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [collab-community/journey-book#13] [Q&A]
What's the Engineering issue template for?

**Comment by Adarsh-jaiss** [comment]
if anyone got a new idea like you have suggested of the website, as I haven't thought of that. so that will help folks to work on a new ide which we haven't discovered. it can increases the boundaries of this project  because we don't know ,where and how it will go in future :)

**Reply by KendallDoesCoding** [reply]
Ok, that can be further explained in the description of the issue thread, I would suggest.

**Reply by Adarsh-jaiss** [reply]
okay so i'm gonna delete that :)

**Reply by KendallDoesCoding** [reply]
Yeah, good idea! People can create suggestions in the feature request template itself.

**Reply by Adarsh-jaiss** [reply]
yeah, that's why I have deleted it.

**Reply by KendallDoesCoding** [reply]
Thanks!
